lib/core.js

```diff
@@ -1,3 +0,0 @@
-'use strict';
-
-module.exports = require('../src/core');
```

Comment:
I guess we'd better bring this file back, as it was [referenced in the readme](https://github.com/plotly/plotly.js/pull/5508/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L82)